spec/models/appointment_summary_spec.rb

```diff
@@ -58,6 +58,15 @@
     expect(subject.format_preference).to eq('standard')
   end
 
+  it do
+    is_expected
+      .to validate_inclusion_of(:appointment_type).in_array(%w(standard 50_54))
+  end
+
+  it 'has a defaults to a standard appointment type' do
```

Comment:
Oops just spotted that too: https://github.com/guidance-guarantee-programme/summary_generator/blob/master/spec/models/appointment_summary_spec.rb#L57